shared/fixes/bash/rpm_verify_hashes.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+
+# Find which files have incorrect hash (not in /etc, because there are all system related config. files) and then get files names
+files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | sed -r 's;^.*\s(.+)$;\1;g' )"
+# From files names get package names and change newline to space, because rpm writes each package to new line
+packages_to_reinstall="$(rpm -qf $files_with_incorrect_hash | tr '\n' ' ')"
+
+if which dnf ; then
```

Comment:
We can use jinja macros and check for `pkg_manager` and avoid this `if` in final remediation snippet.